server/src/models/DogPark.js

```diff
@@ -0,0 +1,27 @@
+const Model = require("./Model.js")
+
+class DogPark extends Model {
+    static get tableName() {
+        return ("dogParks")
+    }
+
+    static get jsonSchema() {
+        return ({
+            type: "object",
+            required: ["name", "description", "address"],
+            properties: {
+                name: {type: "string"},
+                description: {type: "string", minLength: 20 },
+                address: {type: "string"},
+                neighborhood: {type: "string"},
+                hasTrash: {type: "boolean"},
+                hasBags: {type: "boolean"},
+                hasFence: {type: "boolean"},
+                hasWater: {type: "boolean"},
+                hasBenches: {type: "boolean"}
```

Comment:
at the model level, its okay that each of these boolean fields arent required, as long as you make those changes on the db layer that I mentioned! So this file is good :)